sdk/batch/arm-batch/review/arm-batch.api.md

```diff
@@ -1103,7 +1311,7 @@ export interface OutboundEnvironmentEndpointCollection {
 export type PackageState = "Pending" | "Active";
 
 // @public
-export interface Pool extends ProxyResource {
+export interface Pool extends AzureProxyResource {
```

Comment:
a little wired with AzureProxyResource, could you share link with me?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/4883fa5dbf6f2c9093fac8ce334547e9dfac68fa/specification/batch/resource-manager/Microsoft.Batch/stable/2024-07-01/BatchManagement.json#L4004-L4005